manifests/params.pp

```diff
@@ -227,43 +185,10 @@
       $repo_class                = undef
     }
     'Linux': {
```

Comment:
I think this adds little over the default failure

Reply:
Have dropped it.